client/src/CodeClaimPage/ClaimFinished.tsx

```diff
@@ -8,26 +8,35 @@ import { LinkButton } from '../components/LinkButton';
 import ClaimFooterMessage from './ClaimFooterMessage';
 
 /*
-* @dev: Component to show minted token
-* */
-const ClaimFinished: React.FC<{claim: HashClaim}> = ({claim}) => {
+ * @dev: Component to show minted token
+ * */
+
+type Props = {
+  claim: HashClaim;
+};
+
+const ClaimFinished: React.FC<Props> = ({ claim }) => {
   const appLink = `/scan/${claim.beneficiary}`;
+  const isEmailClaimed = claim?.email_claimed;
+
   return (
     <div className={'claim-info'} data-aos="fade-up" data-aos-delay="300">
       <div className={'info-title'}>
-        Congratulations! <br/>
+        Congratulations! <br />
         {claim.event.name} badge is now in your wallet
```

Comment:
Maybe wrapping each string into a `p` tag instead of using `<br />`, even that this code has already been written we should avoid line jumps through `br` tag. Syntactically speaking is more readable two paragraphs vs one line break tag.

```suggestion
        <p>Congratulations!</p>
        <p>{claim.event.name} badge is now in your wallet</p>
```